schema_based_on_diagram.sql

```diff
@@ -1,16 +1,52 @@
-create database clinic
+CREATE DATABASE clinic
 
-use clinic
 
-create table patients(
+CREATE TABLE patients(
   id BIGSERIAL NOT NULL,
     name VARCHAR(100) NOT NULL,
     data_of_birth DATE NOT NULL,
     primary key (id)
-)
+);
 
-create table medicacl_histories(
+CREATE TABLE medicacl_histories(
     id BIGSERIAL NOT NULL,
         admitted_at TIMESTAMP NOT NULL,
-        patients
-)
+       patient_id INT REFERENCES patients(id),
+        status VARCHAR(100) NOT NULL,
+        index (patient_id),
```

Comment:
- Please use the `CREATE INDEX ON medical_histories (patient_id);` to create the index on your FK. please fix this for all tables 🙏